api/nakadi-event-bus-api.yaml

```diff
@@ -351,6 +353,54 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+    delete:
+      tags:
+        - schema-registry-api
+      security:
+        - oauth2: [nakadi.admin]
+      description: |
+        Deletes an `EventType` identified by its name. The underlying Kafka topic and all events of this `EventType`
+        will also be removed.
+        It's important to note that topic deletion in Kafka can take some time, so before it is completelly removed
+        it will not be possible to create an `EventType` with the same name as removed one had (you will get 409
```

Comment:
Are you sure it's gonna return a 409? Because the 409 is a result of a duplicated key in the database. If you already removed the entry from the database, it might have trouble to create the topic but it's going to return a 500.